INSTALL_GUIDE.md

```
@@ -2,10 +2,11 @@
 
 ## Table of Contents
 1. [Obtaining the source code](#obtaining-the-source-code)
-2. [Building project](#building-project)
-3. [Installing Glassfish](#installing-glassfish)
-4. [Preparing the database](#preparing-the-database)
-5. [Deploying the project](#deploying-the-project)
+2. [Installing Maven](#installing-maven)
+3. [Building project](#building-project)
+4. [Installing Glassfish](#installing-glassfish)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)